.travis.yml

```diff
@@ -157,3 +157,14 @@ matrix:
         - mbedtools deploy
         - mbedtools compile -t GCC_ARM -m K64F
         - ccache -s
+
+    - <<: *mbed-tools-test
+      name: "Test deploy command"
+      env: NAME=test-deploy-cmd CACHE_NAME=test-deploy-cmd
+      script:
+        - git clone --depth=1 --single-branch --branch development https://github.com/ARMmbed/mbed-os-example-blinky.git
+        - cd mbed-os-example-blinky
+        - git clone --depth=1 --branch mbed-os-6.1.0 https://github.com/ARMmbed/mbed-os.git
```

Comment:
This test won't work, as we clone only a single branch at depth=1 and then try to fetch other branches. Git doesn't allow that. We'll have to come up with a better test case here.

Reply:
Removed the depth, as we still looking to `mbed deploy` with an empty hash